configure.ac

```diff
@@ -381,6 +381,13 @@ AC_ARG_WITH(nfs-client-package,
             [with_nfs_client_package='false'])
 AC_SUBST(with_nfs_client_package)
 
+AC_ARG_WITH(pcp-package,
+            AC_HELP_STRING([--with-pcp-package=JSON],
+                           [Package that is necessary for PCP, or 'false']),
```

Comment:
Why does this have to be a config option? It's the same name in all OSes that we support, and unlike what the NFS tests do, your tests don't even need a fake package name.

Reply:
You and @mvollmer came to this conclusion in the last round of review, no? Maybe I misunderstood?

Reply:
I don't veto it, and we can do it for the sake of keeping the same pattern. I was just thinking that it's overkill and even harmfull to make our own package name ("cockpit-pcp") configurable, as we control that ourselves. That of course depends on the answer to my question below, whether we should really only install "pcp" or "cockpit-pcp".